freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/FreeIpaV1Endpoint.java

```diff
@@ -52,6 +53,13 @@
             nickname = "listFreeIpaClustersByAccountV1")
     List<ListFreeIpaResponse> list();
 
+    @GET
+    @Path("/health")
```

Comment:
`/` should be removed as the others don't have it. thanks

Reply:
removed